charts/ctlog/values.yaml

```diff
@@ -92,6 +92,10 @@ createctconfig:
   logPrefix: sigstorescaffolding
   privateKeyPasswordSecretName: ""
   ttlSecondsAfterFinished: 3600
+  pubkeysecret: "ctlog-public-key"
```

Comment:
Are these just defaults? Does this value exist in prod?

Reply:
Yeah, these are just the defaults 👍